### PR TITLE
Network: Adds bridge firewall tests

### PR DESCRIPTION
**Pull request by tomponline**
Tests using snap with nftables and xtables drivers.

### CODE REVIEW

**Comment by stgraber** [comment]
@tomponline The test is using a hardcoded device path (/dev/nvmeXYZ) but is run in virtual machines which will not have that.

**Comment by tomponline** [comment]
> @tomponline The test is using a hardcoded device path (/dev/nvmeXYZ) but is run in virtual machines which will not have that.

Oops. Yes I changed that at the last minute from lantea to vm to avoid using lantea unnecessarily. Will change to loopback file.

**Comment by tomponline** [comment]
@stgraber that should fix it now, I've added unmanaged bridge filtering checks too.

File: bin/test-lxd-network-bridge-firewall [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+curl -s http://canonical-lxd.stgraber.org/config/snapd.sh | sh
+
+# Install LXD
+while :; do
+    [ ! -e /usr/bin/lxd ] && break
+    apt remove --purge lxd lxd-client --yes && break
+done
+snap install lxd --edge
+snap install jq
+#snap refresh lxd --channel=latest/edge
```
Comment (stgraber) [inline bin/test-lxd-network-bridge-firewall:31]: This shouldn't be commented

Comment (tomponline) [reply bin/test-lxd-network-bridge-firewall:31]: Doh yep, that was a workaround for the rate limiting from the snap store earlier.

Comment (tomponline) [reply bin/test-lxd-network-bridge-firewall]: fixed

**Pull request merged**